core/src/main/java/org/apache/accumulo/core/conf/Property.java

```diff
@@ -417,6 +417,8 @@
       "The amount of time to sleep between checking for the Montior ZooKeeper lock"),
   MONITOR_LOG_DATE_FORMAT("monitor.log.date.format", "yyyy/MM/dd HH:mm:ss,SSS", PropertyType.STRING, "The SimpleDateFormat string used to configure "
       + "the date shown on the 'Recent Logs' monitor page"),
+  MONITOR_CSS_IMPORT_URLS("monitor.css.import.urls", "", PropertyType.STRING, "Comma separated list of URLs for CSS files to be imported."),
+  MONITOR_JS_IMPORT_URLS("monitor.js.import.urls", "", PropertyType.STRING, "Comma separated list of URLs for JavaScript files to be imported."),
```

Comment:
If it's URLs only that are configurable, then we can't add other HTML attributes to verify integrity of the resource downloaded at those URLs. See https://hacks.mozilla.org/2015/09/subresource-integrity-in-firefox-43/

Also, I'm pretty sure commas are valid characters in a URL.

Reply:
The problem with including a blob of html is how do we determine the default values (without included a big blob of html in Property.java)?  The way I have it here isn't ideal either because if you want to update one version of a JS file, you have to list ALL the external JS files in accumulo-site.

Reply:
And I chose comma since we use that as delimiter in other properties.  You are correct, commas are valid in a URL but I think we could get around by using %2C in the URL.

Reply:
Yeah. Maybe some json syntax?
```javascript
[{"url" : "https://....", "integrity" : "...", "crossorigin" : "anonymous"}, { "url" : "/local/..." }]
```
With "url" being the only required field.

Reply:
Could also just put the default in the template, instead of Property.java, and only use it when the value of the Property is empty.

Reply:
With json, you could add a required "type" field, so you don't need an extra Property. It would just be "monitor.external.resources".

Reply:
> Could also just put the default in the template, instead of Property.java, and only use it when the value of the Property is empty.

That is basically what I am doing now but split up the CSS and JS.  Only putting in 4 URLs to change one, seemed more reasonable than 8.  

Using json is an interesting idea.  Do we currently have a way of parsing it?  I would think Jetty libraries have an easy way...

Reply:
Jackson is a monitor dependency. Gson might be also.